src/js/telemetry/block-change.tsx

```diff
@@ -1,83 +1,95 @@
 /**
  * WordPress dependencies
  */
-// eslint-disable-next-line import/named
-import { BlockInstance } from '@wordpress/blocks';
-import { select, subscribe } from '@wordpress/data';
 import { useEffect } from '@wordpress/element';
+import { select, subscribe } from '@wordpress/data';
+
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import debounce from 'lodash.debounce';
```

Comment:
Should this be `lodash/debounce`, or does it make no difference? Either way, let's stick to one convention (we're using `lodash/isequal` in another PR).